### ISSUE TITLE
Collapsed fieldsets in forms

### ISSUE THREAD

**Issue by Seether85** [douglasmiranda/django-admin-bootstrap#70]
Are they working any differently than in plain Django Admin?
A simple attempt like this:
```
    fieldsets = (
        (None, {
            'fields': ('cliente', 'codice', 'stato', 'data_primo_contatto', 'presenza_software')
        }),
        ('Dettagli commessa', {
            'classes': ('collapse',),
            'fields': ('data_inizio', 'data_fine', 'moduli', 'ore', 'prodotto')
        }),
    )
```
simply makes the whole block completely invisible, although present in html code:
```
<fieldset class="module aligned collapse collapsed">
```
Manually removing the ```collapsed``` class simply makes the form appear as an empty block, but nothing can be read inside.
Could that be a CSS problem maybe?

**Comment by douglasmiranda** [comment]
Did you installed the django-admin-bootstrap==0.3.4?

If not, install the latest version, and clear the cache.

This was fixed: ba789b596ed1c10bbf96829df595a529c3bbb69a

**Comment by Seether85** [comment]
Right. Just updated and flushed some cached statics.
Seems ok now, thank you.
